system/doc/reference_manual/data_types.xml

```diff
@@ -87,6 +87,37 @@
 11> <input>1_234.333_333</input>
 1234.333333
 </pre>
+<marker id="numeric_comparisons" />
+    <section>
+      <title>Comparisons</title>
+      <p>Both integers and floats share the same linear order. That is,
+        <c>1</c> compares less than <c>2.4</c>, <c>3</c> compares greater than
+        <c>2.99999</c>, and <c>5</c> is equal to <c>5.0</c>.</p>
+      <p>When wanting to match a specific number without also matching its
+        integer- or floating-point twin, it may be tempting to use the term
```

Comment:
Nitpick: the negation is hard to process here, a suggestion to special out the specific comparison pairs:

```suggestion
      <p>When wanting to match an integer with another integer or a float
        with another float, it may be tempting to use the term
```